### DISCUSSION TITLE
背景图片更换不成功

### DISCUSSION THREAD

**Discussion by ElegantLee** [YunYouJun/hexo-theme-starry#326] [Q&A 疑难杂症] [answered]
大佬，你做的主题用了挺久的了，感觉非常棒。最近想起来改背景图片，按照文档配置了，但是并没有生效。我将图片放在source/images目录下和使用CDN图片链接都试过了，还是默认的背景图片，不知道是哪里没弄对。。。
![image](https://user-images.githubusercontent.com/45191822/179399361-8d392e8f-19c8-4b75-b0fd-749bcc4a7b15.png)

**Comment by YunYouJun** [comment, marked as answer]
<img width="1633" alt="image" src="https://user-images.githubusercontent.com/25154432/179401917-68e90d4d-f326-44f0-8e85-7cd667210d53.png">

![image](https://user-images.githubusercontent.com/25154432/179401964-6bb05785-bf8a-427a-a2f8-545a86e6ecf7.png)

看起来没有问题，检查版本是否更新？

**Reply by YunYouJun** [reply]
或者可以自己用高斯模糊预处理一下，体积更小，性能更好。
